Makefile

```diff
@@ -40,6 +41,9 @@ deb: build
 	-s dir \
 	-t deb \
 	-v "$(VERSION)" \
+	--conflicts $(OLD_NAME) \
+	--provides $(OLD_NAME) \
```

Comment:
Nope. This is basically to have `libcs50-java` replaces `lib50-java` per https://www.debian.org/doc/debian-policy/ch-relationships.html#s-conflicts.

> A special exception is made for _packages which declare a conflict_ with their own package name, or _with a virtual package which they provide_ (see below): this does not prevent their installation, and _allows a package to conflict with others providing a replacement for it_. You use this feature when you want the package in question to be the only package providing some feature.